README.md

```diff
@@ -219,6 +228,13 @@ Appctl supports multiple ```--env``` variables
 ./appctl deploy --app-name <name> --image <docker-image path> --env key1=value1 --env key2=value2
 ```
 
+Appctl supports variables passed through a env file
+
+```sh
+./appctl deploy --app-name <name> --image <docker-image path> -f <env-file path>
+
+Example:
+./appctl deploy -n hello -i gcr.io/knative-samples/helloworld-go -f /Users/user/variables.env
```

Comment:
- Add an example of how variables.env would look like.
- Add the behaviour when env variables are passed both via file and command line.